cmd/nerdctl/network_inspect_test.go

```diff
@@ -66,3 +66,24 @@ func TestNetworkInspect(t *testing.T) {
 	}
 	assert.DeepEqual(base.T, expectedIPAM, got.IPAM)
 }
+
+func TestNetworkInspectWithNamespace(t *testing.T) {
+	if testutil.GetTarget() == testutil.Docker {
+		t.Skip("test skipped for docker")
+	}
+
+	t.Parallel()
+
+	base := testutil.NewBase(t)
+	baseOther := testutil.NewBaseWithNamespace(t, "nerdctl-other")
+	networkName := testutil.Identifier(t)
+
+	tearDown := func() {
+		base.Cmd("network", "rm", networkName).Run()
```

Comment:
The `nerdctl-other` namespace should be pruned

Reply:
(The namespace name will need to have an extra suffix to avoid racing)

Reply:
- changed from `nerdctl-other` to `tID`.
- added a prune for it in the tearDown function